main.go

```diff
@@ -107,12 +124,12 @@ func initAction(ctx context.Context) (*action.FrizbeeAction, error) {
 
 	// Read the action settings from the environment and create the new frizbee replacers for actions and images
 	return &action.FrizbeeAction{
-		Client:    github.NewClient(tc),
+		Client:    githubClient,
 		Token:     token,
 		RepoOwner: repoOwner,
 		RepoName:  strings.TrimPrefix(repoFullName, repoOwner+"/"),
 
-		ActionsPath:        os.Getenv("INPUT_ACTIONS"),
+		ActionsPaths:       envToStrings("INPUT_ACTIONS"),
```

Comment:
I'd prefer to add a second argument which covers the multiple-paths option (which could be exclusive with the single argument), OR to change `valToStrings` to also interpret the input `foo` as equivalent to `["foo"]`.  (To be honest, I'd rather have had comma-delimited strings over JSON elsewhere, but that ship has sailed.)

Reply:
Thinking further, I converting `foo` to `["foo"]` is dangerous, because we might end up with `["[\"foo\",\n]"` depending on how strict the JSON parser is and how lazy the user is.  Maybe add a second argument for multiple GitHub Actions directories?